logging/cluster-logging-upgrading.adoc

```diff
@@ -1,20 +1,31 @@
 :_content-type: ASSEMBLY
 :context: cluster-logging-upgrading
 [id="cluster-logging-upgrading"]
-= Updating OpenShift Logging
+= Updating Logging
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-[id="cluster-logging-supported-versions"]
-== Supported Versions
-For version compatibility and support information, see link:https://access.redhat.com/support/policy/updates/openshift#logging[Red Hat OpenShift Container Platform Life Cycle Policy]
+There are two types of {logging} updates: minor release updates (5.x.x) and major release updates (5.x).
 
-To upgrade from cluster logging in {product-title} version 4.6 and earlier to OpenShift Logging 5.x, you update the {product-title} cluster to version 4.7 or 4.8. Then, you update the following operators:
+[id="cluster-logging-upgrading-minor"]
+== Minor release updates
 
-* From Elasticsearch Operator 4.x to OpenShift Elasticsearch Operator 5.x
-* From Cluster Logging Operator 4.x to Red Hat OpenShift Logging Operator 5.x
+If you installed the {logging} Operators using the *Automatic* update approval option, your Operators receive minor version updates automatically. You do not need to complete any manual update steps.
 
-To upgrade from a previous version of OpenShift Logging to the current version, you update OpenShift Elasticsearch Operator and Red Hat OpenShift Logging Operator to their current versions.
+If you installed the {logging} Operators using the *Manual* update approval option, you must manually approve minor version updates. For more information, see xref:../operators/admin/olm-upgrading-operators.adoc#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update].
 
-include::modules/cluster-logging-updating-logging-to-current.adoc[leveloffset=+1]
+[id="cluster-logging-upgrading-major"]
+== Major release updates
+
+For major version updates, for example, to update your cluster from version 5.6 to version 5.7, you must complete some manual steps.
```

Comment:
I'm a bit afraid that the wording here might make it unclear about which manual steps are we talking about 🤔 Are we talking about manual steps to upgrade from one major version to the other or are we talking about manual steps to upgrade from 5.6 to 5.7?

Reply:
One major version to another, I was just trying to include an example to make it clearer, but maybe it's clearer without?